src/components/Footer/index.js

```diff
@@ -1,15 +1,270 @@
+      menu.LearningTracks.forEach((item) => {
+        if (pathway.code === item.code) {
+          item["id"] = pathway.id;
+        }
+      });
+    });
+
+  useEffect(() => {
+    if (
+      window.location.pathname.split("/").includes("course-content") ||
+      window.location.pathname.split("/").includes("login") ||
+      window.location.pathname.split("/").includes("profile")
+    ) {
+      console.log("here");
+      setShowHeader(false);
+    }
+  }, []);
+
+  const history = useHistory();
+  history.listen((location, action) => {
+    if (
+      location.pathname.split("/").includes("course-content") ||
+      location.pathname.split("/").includes("login") ||
+      location.pathname.split("/").includes("profile")
+    ) {
+      console.log("not in header");
+      setShowHeader(false);
+    } else {
+      setShowHeader(true);
+    }
+  });
+
   return (
-    <div className="footer">
-      <Link className="footer-link" to={PATHS.PRIVACY_POLICY}>
-        Privacy Policy
-      </Link>
-    </div>
+    <Box
+      style={{
+        display: showHeader ? "inherit" : "none",
+      }}
+      maxWidth="false"
+      bgcolor="primary.light"
+    >
+      <Container maxWidth="xl">
+        <Grid container spacing={2} sx={{ mt: "50px" }}>
+          <Grid xs={12} md={4} sx={{ pl: { sm: 0, md: "16px" } }}>
+            <Box sx={{ display: "flex" }}>
+              <Box className={classes.logo}>
+                <img
+                  src={require("./asset/logo.svg")}
+                  alt="social media"
+                  loading="lazy"
+                />
+              </Box>
+            </Box>
+            <Box className={classes.socialMedia} sx={{ display: "flex" }}>
+              {["facebook", "linkedIn", "twitter"].map((imgName) => (
+                <FooterIcon name={imgName} />
+              ))}
+            </Box>
+            <Box className={classes.content}>
+              <Typography variant="body2" color="text.primary">
+                © 2022 NavGurukul Foundation for Social Welfare
+              </Typography>
+            </Box>
+          </Grid>
+          <Grid xs={6} md={2} sx={{ pl: "15px" }}>
+            {MenuList("About")}
+          </Grid>
+          <Grid xs={6} md={2}>
+            {MenuList("LearningTracks")}
+          </Grid>
+          <Grid xs={6} md={2} sx={{ pl: "15px" }}>
+            {MenuList("GetInvolved")}
+          </Grid>
+          <Grid xs={6} md={2}>
+            <Typography
+              color="text.primary"
+              sx={{ mt: 4, mb: 1 }}
+              variant="subtitle1"
+              component="div"
+            >
+              Learn on Mobile
+            </Typography>
+            <a
+              href="https://play.google.com/store/apps/details?id=org.merakilearn&hl=en_IN&gl=US"
+              target="_blank"
+              rel="external"
+              style={{
+                textDecoration: "none",
+              }}
+            >
```

Comment:
Maybe create an external link component to establish consistency between how these types of links are handled (e.g., opening in a new window):
```
export default function ExternalLink(props) {
  return <a target="_blank" rel="external" {...props} />
}
```

Then you could use the following:
```suggestion
            <ExternalLink 
              href="https://play.google.com/store/apps/details?id=org.merakilearn&hl=en_IN&gl=US"
              style={{
                textDecoration: "none",
              }}
            >
```